src/arch/xtensa/configs/tigerlake_defconfig

```diff
@@ -0,0 +1,6 @@
+CONFIG_TIGERLAKE=y
+CONFIG_CAVS_DMIC=y
+CONFIG_CAVS_SSP=y
```

Comment:
can we add CONFIG_CAVS_ALH?

Reply:
We can when it (sdw) will be merged. Idk if I missed something but I couldn't find that config.